e2e/e2e_test.go

```diff
@@ -146,21 +122,28 @@ func TestCreatesSecureClusterWithGeneratedCert(t *testing.T) {
 	sb := testenv.NewDiffingSandbox(t, env)
 	sb.StartManager(t, controller.InitClusterReconcilerWithLogger(testLog))
 
-	b := testutil.NewBuilder("crdb").WithNodeCount(1).WithTLS().WithEmptyDirDataStore()
+	builder := testutil.NewBuilder("crdb").WithNodeCount(3).WithTLS().
+		WithImage("cockroachdb/cockroach:v20.2.5").
+		WithPVDataStore("1Gi", "standard" /* default storage class in KIND */)
 
 	create := Step{
-		name: "creates 1-node secure cluster",
+		name: "creates 3-node insecure cluster",
```

Comment:
This function is "TestCreatesSecureClusterWithGeneratedCert" but its creating an insecure cluster?

Reply:
typo